app/src/main/res/layout/activity_view_pager.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <RelativeLayout
+        android:id="@+id/view_pager_lyt_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+    <View
```

Comment:
What's the purpose of this `View` ? is it a gap or empty space? a background? 

You could name it `_background` or `_vertical_space` instead of `_view` to give the name extra meaning, and also you avoid `view_pager_view` which sounds strange